scopes/api-reference/sections/api-reference-page/api-reference-page.module.scss

```diff
@@ -0,0 +1,49 @@
+@import '~@teambit/ui-foundation.ui.constants.z-indexes/z-indexes.module.scss';
```

Comment:
seems to be copy pasted from split pane.
can we reuse the original component? or even its shared css?